### PR TITLE
Improvements of UI.

### PR DESCRIPTION
**Pull request by mermetbt**
Hi all,
I'm a new member of XinCheJian and I saw that we are able to fork the project... So that's what I did and I hope you will enjoy that ;-)
Regards,

### CODE REVIEW

File: js/bootstrap.js [hunk 1]
```diff
@@ -0,0 +1,2363 @@
+/*!
```
Comment (lionello) [inline js/bootstrap.js]: Do we really need this file? You only seem to reference bootstrap.min.js

**Comment by lionello** [comment]
Same goes for `bootstrap.css`, only seems to be using the minified version.

**Comment by lionello** [comment]
By the way: thanks! Great to have members contribute to this. Hope we can make it easier and better.

**Comment by mermetbt** [comment]
Hi, in fact I just putted the whole bootstrap package but havn't removed the unused files.

**Comment by lionello** [comment]
Could you remove the unused parts? Ideally, clean out the .css to remove the unreferenced parts. There's tools for that (although I can't remember them now).

**Comment by lionello** [comment]
@rngadam Can you review this and `git pull` in `/var/www` if OK?

**Comment by lionello** [comment]
Guys, don't let this good work left unused.
@rngadam Can you review after deploying?
@mermetbt Can you address my concern, i.e. remove unused files?

**Comment by mermetbt** [comment]
Yes I can remove the unminified files from the repository.

Concerning the unreferenced parts, I never heard about a tools for that, and it's really hard to do with dynamic web pages. The bootstrap framework is a framework, and like all the framework it's not designed to be separated. For futur evolutions, it's better to keep it complete.

**Comment by lionello** [comment]
Looks good!

**Pull request merged**